Content.Server/Voting/VoteCommands.cs

```diff
@@ -69,6 +77,12 @@ public sealed class CreateCustomCommand : IConsoleCommand
         public string Description => Loc.GetString("cmd-customvote-desc");
         public string Help => Loc.GetString("cmd-customvote-help");
 
+        // Webhook stuff 
+        private string _webhookUrl = string.Empty;
+        private string _serverName = string.Empty;
+        private string _avatarUrl = String.Empty;
+        private readonly HttpClient _httpClient = new();
```

Comment:
Should a random command be persisting a HttpClient? Seems like this should go under some generic manager instead.

Reply:
I followed what ahelp did (more or less), though given that's less of a "random command". I _really_ have no clue how to do that but I'm sure I could learn if it's something you would really prefer